test/manipulation.test.js

```diff
@@ -2366,7 +2366,7 @@ describe('manipulation', function() {
     it('should not coerce invalid values provided in where conditions', function(done) {
       Person.update({name: 'Brett Boe'}, {dob: 'notadate'}, function(err) {
         should.exist(err);
-        err.message.should.equal('Invalid date: notadate');
+        err.message.should.equal('Invalid date: Invalid Date');
```

Comment:
FYI this change is because we coerce date values in update now, and then validate them using `Model._coerce` in https://github.com/strongloop/loopback-datasource-juggler/pull/1702/files#diff-d717a01e40d8164976e4468af1bce663R2330. I was going to return the invalid string value for the date instead in the first pass (during model instantiation), but we have other tests which expect `Invalid Date` to be returned for those values (https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/manipulation.test.js#L2226), and I feel like that'd not be backwards compatible as opposed to a not so helpful message here.